plasmapy/utils/calculator/main_interface.py

```diff
@@ -22,6 +23,12 @@
     _process_queue,
 )
 
+warnings.filterwarnings(
+    action="ignore",
+    message="Passing unrecognized arguments",
+    category=DeprecationWarning,
+)  # see issue 2370 for a deprecation warning from traitlets
+
```

Comment:
Strangely, we were getting deprecation warnings from this file when running tests, even though this file is not run during tests (I think).

Reply:
Hmmmmmm?